scripts/build/Jenkins/Jenkinsfile

```diff
@@ -237,25 +313,20 @@ def CheckoutBootstrapScripts(String branchName) {
             [$class: 'CloneOption', noTags: false, reference: '', shallow: false]
         ],
         submoduleCfg: [],
-        userRemoteConfigs: scm.userRemoteConfigs
```

Comment:
Looks like AR is failing at this step `CheckoutEngineBootstrapScripts()`. This works fine in a branch, but failing on this PR. There may be a setting missing from the generated `userRemoteConfigs` that was provided through the built-in `scm.userRemoteConfigs`. Possibly the refspec?